pymongo/srv_resolver.py

```diff
@@ -48,11 +48,11 @@ def _resolve(*args, **kwargs):
 
 class _SrvResolver(object):
     def __init__(self, fqdn,
-                 connect_timeout, srv_service_name):
+                 connect_timeout, srv_service_name, srv_max_hosts):
         self.__fqdn = fqdn
         self.__srv = srv_service_name
         self.__connect_timeout = connect_timeout or CONNECT_TIMEOUT
-
+        self.__srv_max_hosts = srv_max_hosts
```

Comment:
This should be ` srv_max_hosts or 0` so that we don't have to deal with `self.__srv_max_hosts is not None ` checks.

Reply:
Done.